mlrun/api/api/endpoints/tags.py

```diff
@@ -71,8 +71,8 @@ async def overwrite_object_tags_with_tag(
 @router.put("/projects/{project}/tags/{tag}", response_model=mlrun.api.schemas.Tag)
 async def append_tag_to_objects(
     project: str,
-    tag: str,
-    tag_objects: mlrun.api.schemas.TagObjects,
+    tag: str = fastapi.Path(..., regex=tag_name_regex_as_string()),
+    tag_objects: mlrun.api.schemas.TagObjects = fastapi.Body(...),
```

Comment:
what does that default means?

Reply:
This is a body placeholder.
All cases are enforced by pydantic:
1. If tag object is None, the result is 422.
2. if tag object does not contain one of the fields (kind or identifiers), the result is 422
3. If tag object contains additional fields, they will be ignored and will not be passed to the function.

So we're covered, and the fastapi.Body with ellipsis is just a placeholder for a default value that will never be used.